### ISSUE TITLE
Enabling TOTP

### ISSUE THREAD

**Issue by baughmann** [boschkundendienst/guacamole-docker-compose#45]
Firstly, thank you so much for posting this. You've made countless lives easier, and I greatly appreciate it.

Is there a straight-forward way to enable TOTP / MFA off the bat here? I tried throwing in some environment variables in the `docker-compose-yaml` but after reading the Guacamole docs it seems like it might end up being a bit more involved than that.

Any pointers? Thanks in advance.

**Comment by boschkundendienst** [comment]
Hi, afaik it should be just as simple as adding the correct environment variables. Additionally you might have to map GUACAMOLE_HOME to some local dir and then the start.sh script within the container will then copy files into GUACAMOLE_HOME automatically and your configurations will persist across container restarts.

Anyways, not part of this repo. If you find a good howto feel free to attach it to this issue.

**Comment by megacorvega** [comment]
I got this to work, and it should be persistent across restarts, but I'm sure someone with a bit more acumen can clean it up. I added the following to the gucamole service in `docker-compose.yml`

```
environment:
    GUACAMOLE_HOME: /ghome
volumes:
    - ./home:/ghome:rw
``` 

Then I edited the prepare script to include `mkdir ./home/extensions >/dev/null 2>&1`. From there, just copy in whatever .jar extension you want to use to the `extensions` folder, and it should be good to go!
